src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -0,0 +1,575 @@
+--!strict
+
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local MarketplaceService = game:GetService("MarketplaceService")
+
+local Packages = ReplicatedStorage.Packages
+local Knit = require(Packages.Knit)
+local GameAnalytics = require(Packages.GameAnalytics)
+
+local CURRENCIES = { "Coins" } -- List of all in-game currencies
+local BUILD = "0.0.1" -- Game version
```

Comment:
Should the build variable be a static constant? I feel that the build should be set during the init phase of the analytics service by another external script that tells it what the current build is.